apps/framework-cli-e2e/test/utils/logger.ts

```diff
@@ -0,0 +1,202 @@
+/**
+ * E2E Test Logger
+ * Provides structured logging with local timestamps for e2e tests
+ */
+
+export enum LogLevel {
+  DEBUG = 0,
+  INFO = 1,
+  WARN = 2,
+  ERROR = 3,
+}
+
+export interface LoggerConfig {
+  level: LogLevel;
+  useColor: boolean;
+}
+
+class Logger {
+  private config: LoggerConfig;
+
+  constructor(config: Partial<LoggerConfig> = {}) {
+    this.config = {
+      level: config.level ?? LogLevel.INFO,
+      useColor: config.useColor ?? true,
+    };
+  }
+
+  /**
+   * Get current timestamp in local time format
+   */
+  private getTimestamp(): string {
+    const now = new Date();
+    const year = now.getFullYear();
+    const month = String(now.getMonth() + 1).padStart(2, "0");
+    const day = String(now.getDate()).padStart(2, "0");
+    const hours = String(now.getHours()).padStart(2, "0");
+    const minutes = String(now.getMinutes()).padStart(2, "0");
+    const seconds = String(now.getSeconds()).padStart(2, "0");
+    const ms = String(now.getMilliseconds()).padStart(3, "0");
+
+    return `${year}-${month}-${day} ${hours}:${minutes}:${seconds}.${ms}`;
+  }
```

Comment:
this can't be the best way to do this? Can't we pass in a c timestamp string and use that?